common/homeConfig.ts

```diff
@@ -0,0 +1,3 @@
+export type HomeConfig = {
```

Comment:
Is there a better place to store this? It feels sort of pointless to create a new root collection just for this, but I don't know where else we'd put it. I thought about adding a "homepage" group and using the existing "pinnedItems" field from that group, but it seems too hacky. I thought about adding it to some other existing root collection, but none of them really fit conceptually ('stats' is the only static one, but it doesn't make a lot of sense).